tox.ini

```diff
@@ -23,12 +23,13 @@ python =
     3.6: py36
     3.7: py37
     3.8: py38
-    3.9: py39
+    3.9-dev: py39
 unignore_outcomes = True
 
 [travis:env]
 DJANGO =
     2.2: django22
     3.0: django30
     3.1: django31
+    3.2: django32
```

Comment:
There is no Django 3.2 logic above in `[testenv]` `deps`

Reply:
Good catch thanks, will fix tomorrow

Reply:
On top, my bad. I thought they released it today, but nope, we are not April 2021 (yet).